modules/ROOT/pages/architecture.adoc

```diff
@@ -85,6 +85,11 @@ image::architecture-network.png[]
 
 This diagram shows the TCP load balancer used to load balance requests across the internal load balancers running on the controllers. It also shows the internal load balancer that distributes requests to each replica of Mule applications running on the workers.
 
+For internal load balancers:
+
+* *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
```

Comment:
GLOBALLY adopt the style reference admonition to limit list items to one fragment (or sentence) max, with additional text in a plain para below:
```suggestion
* *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. 
In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
```
Does "with the amount of CPU and memory specified" modify "runs" or "node"?